global.json

```diff
@@ -21,7 +21,7 @@
     "rollForward": "latestPatch"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24170.6",
-    "Microsoft.Build.NoTargets": "3.7.0"
+    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24266.3",
+    "Microsoft.Build.NoTargets": "2.0.1"
```

Comment:
@NikolaMilosavljevic - Is this going to impact source-build?  Will this get lifted to 3.7.0?

Reply:
@MichaelSimons this came about from resolving a conflict when merging against `main`. If the other is the better option let me know. Will correct in a future PR. Need to get this resolved ot unblock a VS razor issue.

Reply:
It won't impact source-build. We always use the latest `NoTargets` SDK.